package.json

```diff
@@ -63,6 +63,7 @@
     "grunt-exec": "0.4.6",
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
+    "grunt-cli": "0.1.13",
```

Comment:
That sounds reasonable. We're coming from a place where all COINS developers' global NPM packages are managed for them by Ansible, so they have never been missing `grunt-cli`. Adding it to the README seems like the best idea.